CHANGELOG.md

```diff
@@ -44,7 +44,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 - [#226](https://github.com/thanos-io/kube-thanos/pull/226) Only schedule thanos components on linux nodes.
 
 ### Added
-
+- [#244](https://github.com/thanos-io/kube-thanos/pull/244) Add functions to implement Thanos Receive split proposal.
```

Comment:
It's not a proposal anymore we actually have it in upstream :)

Reply:
Haha good point - updated.